### ISSUE TITLE
ipv6

### ISSUE THREAD

**Issue by lbtboy** [AdguardTeam/AdGuardHome#4708]
AdGuard Home v.0.107.7

Hello. Is it possible to somehow disable ipv6 requests for one device on the local network or at least for one site?

**Comment by fernvenue** [comment]
Of course, you can disable anything you don't want by custom filtering rule, here are examples:

- Disable IPv6 resolve for `127.0.0.1`:

```
*$dnstype=AAAA,dnsrewrite=NOERROR,client='127.0.0.1'
```

- Disable specific site's IPv6 for `127.0.0.1`:

```
||example.com^$dnstype=AAAA,dnsrewrite=NOERROR,client='127.0.0.1'
```

You can also use `NXDOMAIN`, and `SERVFAIL` as response code, but I strongly suggest you to use `NOERROR` here, this will prevent the client from retrying the resolution with another DNS server.

For more information you can read AdGuardHome's wiki, [Hosts Blocklists](https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists).

**Comment by lbtboy** [comment]
None of the rules work.

**Comment by fernvenue** [comment]
> None of the rules work.

They all work fine, but are just examples, follow these rules and write your own.

Test `||icanhazip.com^$dnstype=AAAA,dnsrewrite=NOERROR,client='127.0.0.1'`:

- Before:

```
~# nslookup icanhazip.com
Server:         127.0.0.1
Address:        127.0.0.1#53

Non-authoritative answer:
Name:   icanhazip.com
Address: 104.18.114.97
Name:   icanhazip.com
Address: 104.18.115.97
Name:   icanhazip.com
Address: 2606:4700::6812:7261
Name:   icanhazip.com
Address: 2606:4700::6812:7361
```

- After:

```
~# nslookup icanhazip.com
Server:         127.0.0.1
Address:        127.0.0.1#53

Non-authoritative answer:
Name:   icanhazip.com
Address: 104.18.114.97
Name:   icanhazip.com
Address: 104.18.115.97
```

I think you really need to read the documentation first, and use clear headings next time.

**Comment by lbtboy** [comment]
> I think you really need to read the documentation first, and use clear headings next time.

I advise you to do the same, since these rules do not work, These rules only work on the device where AdGuardHome (router) is installed, but these rules do not work for my device (smarttv) on the local network. *$dnstype=AAAA,dnsrewrite=NOERROR,client='127.0.0.1' this rule also does not work for the local device 192.168.1.48. Do I understand correctly that for the device 192.168.1.48 it should look like this *$dnstype=AAAA,dnsrewrite=NO ERROR,client='192.168.1.48'?

**Comment by fernvenue** [comment]
> > I think you really need to read the documentation first, and use clear headings next time.
> 
> I advise you to do the same, since these rules do not work, These rules only work on the device where AdGuardHome (router) is installed, but these rules do not work for my device (smarttv) on the local network. *$dnstype=AAAA,dnsrewrite=NOERROR,client='127.0.0.1' this rule also does not work for the local device 192.168.1.48. Do I understand correctly that for the device 192.168.1.48 it should look like this *$dnstype=AAAA,dnsrewrite=NO ERROR,client='192.168.1.48'?

Of course, so I told you over and over again that they are just examples, that means you should write your own rules. Just imagine, what is the probability that a stranger will be able to guess the IP when you are not saying which IP you want to block? And, even if you're not sure what you're thinking, why not just try it?

Again, this should go to the [discussions](https://github.com/AdguardTeam/AdGuardHome/discussions), and you really need to read the documentation first, and use clear headings next time. If everyone asks questions in the issues that can be easily solved with documentation, it will put a huge burden on the development team.

**Comment by lbtboy** [comment]
> > > I think you really need to read the documentation first, and use clear headings next time.
> > 
> > 
> > I advise you to do the same, since these rules do not work, These rules only work on the device where AdGuardHome (router) is installed, but these rules do not work for my device (smarttv) on the local network. *$dnstype=AAAA,dnsrewrite=NOERROR,client='127.0.0.1' this rule also does not work for the local device 192.168.1.48. Do I understand correctly that for the device 192.168.1.48 it should look like this *$dnstype=AAAA,dnsrewrite=NO ERROR,client='192.168.1.48'?
> 
> Of course, so I told you over and over again that they are just examples, that means you should write your own rules. Just imagine, what is the probability that a stranger will be able to guess the IP when you are not saying which IP you want to block? And, even if you're not sure what you're thinking, why not just try it?
> 
> Again, this should go to the [discussions](https://github.com/AdguardTeam/AdGuardHome/discussions), and you really need to read the documentation first, and use clear headings next time. If everyone asks questions in the issues that can be easily solved with documentation, it will put a huge burden on the development team.

Oh, yes, that's all you can do, give a simple excuse and send away! What examples have I made it clear to you that I need all requests to be blocked from the address 192.168.1.48, that's all, but you start to get smart and shine with a narrow mind. All the best thanks for not refusing to help.
